### ISSUE TITLE
Delete images from disk

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#181]
```
Feature Request:

I noticed that keepnote is not deleting images from disk if they are
deleted inside keepnote.
Sometimes this is desired (e.g. if a picture is also linked elsewhere).
Sometimes this leads to garbage data because deep in a subfolder the
"deleted" image remains unwanted...

This problem could perfectly be resolved e.g. by an addition context entry:
"delete from disk" which not only deletes the image inside keepnote from a
page but also wipes it from the disk.

still a wounderful program :)   thanx!

[Windows XP, Vista, Win7; keepnote 0.6.1 (03.01.2010)]



```

Original issue reported on code.google.com by `nudels0...@googlemail.com` on 12 Jan 2010 at 7:51

**Comment by GoogleCodeExporter** [comment]
Original comment by `matt.rasmus@gmail.com` on 15 Jan 2010 at 4:37
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I share this feature request. An alternative approach might be a "tidy up" 
function
where notes are scanned for their links to check they're still valid. At the 
same
time they are compared with files inside the current workbook tree to remove
unreferenced files.

Thanks for writing KN!
[Ubuntu JJ; keepnote 0.6.1]
```

Original comment by `elfan...@gmail.com` on 17 Jan 2010 at 9:32

**Comment by GoogleCodeExporter** [comment]
```
I support this suggestion. delete from disk would be important.
```

Original comment by `zhisheng...@qq.com` on 1 Sep 2010 at 5:38

**Comment by GoogleCodeExporter** [comment]
```
This is now implemented in keepnote 0.6.8.
```

Original comment by `matt.rasmus@gmail.com` on 23 Feb 2011 at 3:47
- Changed state: **Done**
- Added labels: **notebook**
